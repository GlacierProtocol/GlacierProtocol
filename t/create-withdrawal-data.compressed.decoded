{
  "txid": "4dee3bc081dda755878339c4cfe222cf1d29ec7af5e7cd19a746b50623c9ec8d",
  "hash": "4dee3bc081dda755878339c4cfe222cf1d29ec7af5e7cd19a746b50623c9ec8d",
  "version": 2,
  "size": 405,
  "vsize": 405,
  "weight": 1620,
  "locktime": 0,
  "vin": [
    {
      "txid": "05fbb9ed04a7d06c8504723c40800b65f097b34fd0a5b85e0c9cc8457974e5a8",
      "vout": 1,
      "scriptSig": {
        "asm": "0 304402206634c1e8fbebddb4263da22afb7631007b7cab48cd8da6b9e7e4868e635d86d502207505455695b0bb1d26838eaebcaf8dc6af46ac872a7f9fc8ee53cca2c6e68aee[ALL] 3044022015292b64da00689a857be53be71a0bcb5cc6a269cf74435d871c4a6482bb4640022069df9a1eeead1ee4fd7dcb604be55e5eb42e93d2e61b7451e067309b4c8d5a9a[ALL] 52210277f15f22aeffaf3f3bc48a280a767f7c6af21276783c09ff3bcaabbece178113210203f642223bc751e125ab604ed20c544e933eb82a19201f90bda6236a9ddbedac21026895f3ed01c09025ba9cfab7b67d699e3ebaceada0beff70bc72c7f10df2d1c621039f3cb152130c241e01b9220a147f69f7111b830def35b510973ab47bb9af7ff654ae",
        "hex": "0047304402206634c1e8fbebddb4263da22afb7631007b7cab48cd8da6b9e7e4868e635d86d502207505455695b0bb1d26838eaebcaf8dc6af46ac872a7f9fc8ee53cca2c6e68aee01473044022015292b64da00689a857be53be71a0bcb5cc6a269cf74435d871c4a6482bb4640022069df9a1eeead1ee4fd7dcb604be55e5eb42e93d2e61b7451e067309b4c8d5a9a014c8b52210277f15f22aeffaf3f3bc48a280a767f7c6af21276783c09ff3bcaabbece178113210203f642223bc751e125ab604ed20c544e933eb82a19201f90bda6236a9ddbedac21026895f3ed01c09025ba9cfab7b67d699e3ebaceada0beff70bc72c7f10df2d1c621039f3cb152130c241e01b9220a147f69f7111b830def35b510973ab47bb9af7ff654ae"
      },
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.00000550,
      "n": 0,
      "scriptPubKey": {
        "asm": "OP_HASH160 606fb72624603795d73b911a729d88f95f9b6ac8 OP_EQUAL",
        "hex": "a914606fb72624603795d73b911a729d88f95f9b6ac887",
        "reqSigs": 1,
        "type": "scripthash",
        "addresses": [
          "2N238gpicqPFfMz8sUN7pqqaJnYkgsC1WmL"
        ]
      }
    },
    {
      "value": 0.00000400,
      "n": 1,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 5d1e4870ab9df56432aa357021fd0a15a7094ad1 OP_EQUALVERIFY OP_CHECKSIG",
        "hex": "76a9145d1e4870ab9df56432aa357021fd0a15a7094ad188ac",
        "reqSigs": 1,
        "type": "pubkeyhash",
        "addresses": [
          "mp1KQzUoDf9div7jjgon3UuqLThMSo3tmF"
        ]
      }
    }
  ]
}
