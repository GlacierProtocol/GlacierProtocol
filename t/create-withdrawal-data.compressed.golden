Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
You will need to enter several pieces of information to create a withdrawal transaction.


*** PLEASE BE SURE TO ENTER THE CORRECT DESTINATION ADDRESS ***


Source cold storage address: 
Redemption script for source cold storage address: 
Destination address: 
How many unspent transactions will you be using for this withdrawal? 
Please paste raw transaction #1 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Transaction data found for source address.
TOTAL unspent amount for this raw transaction: 0.00005000 BTC

How many private keys will you be signing this transaction with? 
#: Key #1: Key #2: 
Enter fee rate.
Satoshis per vbyte: 
Based on the provided rate, the fee will be 0.00000405 bitcoin.
Confirm? (y/n): 
Fee calculation aborted. Starting over...

Enter fee rate.
Satoshis per vbyte: 
Based on the provided rate, the fee will be 0.00004050 bitcoin.
Confirm? (y/n): 
Please enter the decimal amount (in bitcoin) to withdraw to the destination address.

Example: For 2.3 bitcoins, enter "2.3".

After a fee of 0.00004050, you have 0.00000950 bitcoins available to withdraw.

*** Technical note for experienced Bitcoin users:  If the withdrawal amount & fee are cumulatively less than the total amount of the unspent transactions, the remainder will be sent back to the same cold storage address as change. ***

Amount to send to mp1KQzUoDf9div7jjgon3UuqLThMSo3tmF (leave blank to withdraw all funds stored in these unspent transactions): 0.00000550 being returned to cold storage address address 2N238gpicqPFfMz8sUN7pqqaJnYkgsC1WmL.

Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.00005000 BTC in unspent supplied transactions
0.00000550 BTC going back to cold storage address 2N238gpicqPFfMz8sUN7pqqaJnYkgsC1WmL
0.00000400 BTC going to destination address mp1KQzUoDf9div7jjgon3UuqLThMSo3tmF
Fee amount: 0.00004050

Signing with private keys: 
cSfvoUsWWZ81M4qGYciPHdNxHXQxPEsXBdGWjYdchpisPabtiKVQ
cPWVKeCBjnv833Kayc6nczBbG4YqqbCucup3n5D68WPzBEfceBAQ


Confirm? (y/n): 
Calculating transaction...


Sufficient private keys to execute transaction?
True

Raw signed transaction (hex):
0200000001a8e5747945c89c0c5eb8a5d04fb397f0650b80403c7204856cd0a704edb9fb0501000000fd1e010047304402206634c1e8fbebddb4263da22afb7631007b7cab48cd8da6b9e7e4868e635d86d502207505455695b0bb1d26838eaebcaf8dc6af46ac872a7f9fc8ee53cca2c6e68aee01473044022015292b64da00689a857be53be71a0bcb5cc6a269cf74435d871c4a6482bb4640022069df9a1eeead1ee4fd7dcb604be55e5eb42e93d2e61b7451e067309b4c8d5a9a014c8b52210277f15f22aeffaf3f3bc48a280a767f7c6af21276783c09ff3bcaabbece178113210203f642223bc751e125ab604ed20c544e933eb82a19201f90bda6236a9ddbedac21026895f3ed01c09025ba9cfab7b67d699e3ebaceada0beff70bc72c7f10df2d1c621039f3cb152130c241e01b9220a147f69f7111b830def35b510973ab47bb9af7ff654aeffffffff02260200000000000017a914606fb72624603795d73b911a729d88f95f9b6ac88790010000000000001976a9145d1e4870ab9df56432aa357021fd0a15a7094ad188ac00000000

Transaction fingerprint (md5):
386a08a6e34f806906ab9263bafbde4c
QR code for transaction written to transaction.png
